validator/python/copy.py

```diff
@@ -1,433 +0,0 @@
-"""Generic (shallow and deep) copying operations.
```

Comment:
Heh. It's not even our own implementation. It's just a copy from the standard library. I don't even know why we have it, since I don't think we've ever run a version of Python old enough not to have it.